pytd/client.py

```diff
@@ -202,7 +202,7 @@ def query(self, query, engine=None, **kwargs):
             - ``wait_callback`` (function): called every interval against job itself
             - ``engine_version`` (str): run query with Hive 2 if this parameter
               is set to ``"experimental"`` and ``engine`` denotes Hive.
```

Comment:
It's totally out of cope of this PR, but I wonder `experimental` is still the right option for Hive 2 or not, because this document point `stable` for Hive 2
https://docs.treasuredata.com/display/public/PD/Writing+Hive+Queries

Reply:
I checked with @tiana528 and it turns out Hive 2 is now GA so that setting `engine_version = 'stable'` would be appropriate. `experimental` is limited to specific customers for now.

Reply:
Good catch. You're right - Hive 2 isn't "experimental" anymore. 

How about this 89f1a00a153bdb52562d101d88d03230d5597132? Our documentation site must be up-to-date in terms of version availability, and hence I just avoided mentioning a specific version that is available today.

Reply:
I confirmed woth @johan-g that it'd be better to just replace from `experimental` to `stable`.

Here is the note for current behavior (I hope it'll be on TD doc near furture).

- By default, default Hive version per account is used. For most accounts, it is 0.13 at this moment.
- Setting `stable` enforce to use Hive 2 explicitly

Reply:
3035c5ec93032b2782b1e99216788ca83348fdbd